ui/packages/app/src/Blocks.svelte

```diff
@@ -204,6 +204,7 @@
 			return dep.filter((_id) => _id !== id);
 		});
 	}
+	console.log("Blocks.svelte Hit")
```

Comment:
Leftover comment.

Reply:
Oh sorry was supposed to close this PR. as it doesn't actually solve the issue.